app/assets/stylesheets/_forms.scss

```diff
@@ -0,0 +1,42 @@
+@import "bootstrap";
```

Comment:
Right now yes. But I should put it in a directory just like you did. To prevent this.